public/projects/hangman.md

```diff
@@ -0,0 +1,11 @@
+# Hangman
+
+Based on an [vintage BASIC game](http://www.vintage-basic.net/games.html)  -- see the source code at [HANGMAN.BAS](https://www.atariarchives.org/basicgames/showpage.php?page=80) (but don't try too hard to understand it; it's very dense).
```

Comment:
I like this project for teaching about Hashes and Tries.
Looks like this is missing some detail.
Dave Thomas has a nice progression in a project he designed we can build off of.
https://codestool.coding-gnome.com/courses/elixir-for-programmers